TYPO3.Flow/Classes/TYPO3/Flow/Http/Request.php

```diff
@@ -71,7 +91,24 @@ class Request extends AbstractMessage
      */
     public function __construct(array $get, array $post, array $files, array $server)
     {
+        if (self::$trustedProxiesSettings === null) {
+            self::$trustedProxiesSettings = [
+                'proxies' => '*',
+                'headers' => [
+                    self::HEADER_CLIENT_IP => 'X-Forwarded-For',
+                    self::HEADER_HOST => 'X-Forwarded-Host',
+                    self::HEADER_PORT => 'X-Forwarded-Port',
+                    self::HEADER_PROTOCOL => 'X-Forwarded-Proto'
+                ]
+            ];
+            if (Bootstrap::$staticObjectManager !== null) {
+                $configurationManager = Bootstrap::$staticObjectManager->get(ConfigurationManager::class);
+                self::$trustedProxiesSettings = $configurationManager->getConfiguration(ConfigurationManager::CONFIGURATION_TYPE_SETTINGS, 'TYPO3.Flow.http.trustedProxies');
```

Comment:
So you set the `$trustedProxiesSettings` in 95ff and then override it completely from settings here!?
Wouldn't it make sense to do either one or the other or *merge* the configured ones over the defaults?